api/internal/filter/authentication.go

```diff
@@ -20,11 +20,11 @@ import (
 	"net/http"
 	"strings"
 
-	"github.com/dgrijalva/jwt-go"
 	"github.com/gin-gonic/gin"
 
-	"github.com/apisix/manager-api/conf"
-	"github.com/apisix/manager-api/log"
+	"github.com/apisix/manager-api/internal/conf"
+	"github.com/apisix/manager-api/internal/log"
+	jwt "github.com/dgrijalva/jwt-go"
```

Comment:
I think we should classify the import package.

See https://github.com/uber-go/guide/blob/master/style.md#import-group-ordering

Reply:
Done